addons/share-through-mystuff/userscript.js

```diff
@@ -0,0 +1,25 @@
+export default async function ({ addon, global }) {
+  const share_function = document.createElement("a");
```

Comment:
Using the word function in an object name is a little confusing. Could you use a different variable name and use `camelCase`?